source/core/constraint/unique.d

```diff
@@ -328,87 +295,77 @@ unittest
 	UniqueConstraint uc2 = cells1[2].get!UniqueConstraint; // [2,3]
 
 	assertFalse(uc0.cells.equal(uc1.cells));
-	assertTrue(uc2.cells == [cells1[0],cells1[1]]);
+	assertTrue(uc2.cells.equal([cells1[0],cells1[1]]));
```

Comment:
Shouldn't you use `assertEquals(...)`?

Reply:
I want to make sure every content is equal in each index. Although `==` and `AssertEquals` would work in this case, it's not explicit enough, imo. The better solution would be `AssertArrayEquals`, however you cannot use this function with objects as it is marked `@safe`, and it was marked for refactoring as well as other assertion functions. When the refactoring is completed I'll change this to `AssertArrayEquals`.